pdk/1.1/modules/ROOT/pages/policies-pdk-upgrade-pdk.adoc

```diff
@@ -0,0 +1,103 @@
+= Upgrading PDK
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Flex Gateway Policy Development Kit (PDK) is composed of the following separate versioned components:
+
+* Anypoint CLI PDK plugin
+* PDK Rust libraries
+* Anypoint Cargo plugin
+
+When you create a new custom policy project, the PDK Rust libraries and Anypoint Cargo plugin versions are the same as the Anypoint CLI PDK plugin used to create the project.
```

Comment:
Is this topic about upgrading PDK or the custom policy project?

Reply:
Both?

I tried to make it make more sense in my edits.